src/vitepress/components/VPDocOutlineItem.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
-import { useData, _escapeHtml } from 'vitepress'
+import { decode } from 'tiny-decode'
```

Comment:
```suggestion
import { decode } from 'tiny-decode/browser'
```

Should it be the `browser` export?

Reply:
... and the `decode(text)` might be possibly cached to avoid redundant DOM consumptions. WDYT?

Reply:
It is SSR'd so no. Can't use /browser import. We can add it in computed though.

Reply:
Well, if it's not the browser export, then I think the actual bundle would be larger than you have pasted, right? (The default export includes the `entities` package if I'm not wrong.)

The caching I originally mentioned was about the `decodeHTML5()` function in the browser export https://github.com/natemoo-re/tiny-decode/blob/main/browser.mjs#L4 which just cache by chars.

Reply:
> then I think the actual bundle would be larger than you have pasted

No, vite/rollup use exports conditions. They will bundle the browser export for browser and the import export for ssr (node). (Read the project's readme and package.json.) You can also run `pnpm demo-build` and open `demo/.vitepress/dist/assets/chunks/theme.<hash>.js` and search for decode.

Reply:
If you want to see the intermediate SSR output, run `DEBUG=true pnpm demo-build` and open `demo/.vitepress/.temp/app.js` which will have some code like this:

```ts
import { decode } from "tiny-decode";

        _push(`<li data-v-0b1132c6><a class="outline-link"${ssrRenderAttr("href", link2)} style="${ssrRenderStyle(!hidden ? null : { display: "none" })}" data-v-0b1132c6>${ssrInterpolate(unref(decode)(text))}</a>`);
```

-- so here node will resolve `tiny-decode` to `tiny-decode/node.mjs`

Reply:
Sounds great! Thanks for all the info.